src/pooling_api.cpp

```diff
@@ -41,30 +41,37 @@ inline void Pooling_logging_cmd(const miopenPoolingDescriptor_t poolDesc,
     {
         auto tensor_dim = miopen::deref(tensorDesc).GetSize();
         std::stringstream ss;
-        if(miopen::deref(tensorDesc).GetType() == miopenHalf)
-        {
-            ss << "poolfp16";
-        }
-        else
-        {
-            ss << "pool";
-        }
-        if(tensor_dim == 5)
+
+        switch(miopen::deref(tensorDesc).GetType())
```

Comment:
[2022-07-26T19:02:47.992Z] /home/jenkins/workspace/MIOpen_ddu_fix_pooling_cmd_print/src/pooling_api.cpp:45:16: error: 5 enumeration values not explicitly handled in switch: 'miopenInt32', 'miopenInt8', 'miopenInt8x4'... [clang-diagnostic-switch-enum,-warnings-as-errors]

[2022-07-26T19:02:47.992Z]         switch(miopen::deref(tensorDesc).GetType())

[2022-07-26T19:02:47.992Z]                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

[2022-07-26T19:02:47.992Z] src/CMakeFiles/tidy-target-MIOpen-pooling_api_cpp.dir/build.make:57: recipe for target 'src/CMakeFiles/tidy-target-MIOpen-pooling_api_cpp' failed